rules/php73/tests/Rector/FuncCall/RemoveMissingCompactVariableRector/Fixture/compact_over_array.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Php73\Tests\Rector\FuncCall\RemoveMissingCompactVariableRector\Fixture;
+
+function run()
+{
+    $foo = 'foo';
+    $bar = 'bar';
+
+    compact(['foo', 'bar'], ['baz']);
+    compact(['foo'], ['bar', 'baz']);
+    compact(['foo'], ['bar'], 'baz');
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Php73\Tests\Rector\FuncCall\RemoveMissingCompactVariableRector\Fixture;
+
+function run()
+{
+    $foo = 'foo';
+    $bar = 'bar';
+
+    compact(['foo', 'bar']);
+    compact(['foo'], ['bar']);
+    compact(['foo'], ['bar']);
```

Comment:
There are too many cases it's hard for me to get it.
Could you decouple the simplest case to a new fixture file?